### PR TITLE
changes to who we are

### PR DESCRIPTION
**Pull request by EllaNancyFay**

### CODE REVIEW

File: team.md [hunk 1]
```diff
@@ -6,8 +6,6 @@ redirect_from:
 
 ## Who we are
 
-Red56 are a team of business-focused technologists based mostly in London.
```
Comment (EllaNancyFay) [inline team.md]: This is a repetition of the home page

Comment (timdiggins) [reply team.md]: Yes, actually this was deliberate! (this is an expansion of the home page -- might get there from some other route).

Comment (timdiggins) [reply team.md]: can you put this back

Comment (EllaNancyFay) [reply team.md]: it's back

**Pull request merged**